ui/forms/MainWindow.qml

```diff
@@ -127,34 +128,48 @@ ApplicationWindow {
     text: `Zoom: ${(systemPreview.zoom * 100).toFixed(0)}%`
   }
 
-  Button {
+  RowLayout {
     anchors.bottom: zoomLabel.top
     anchors.left: zoomLabel.left
     anchors.bottomMargin: Theme.padding
-    text: "Debug Source"
-    outlined: true
 
-    onClicked: {
-      astInspectorLoader.active = true;
-    }
+    spacing: Theme.units(4)
 
-    Loader {
-      id: astInspectorLoader
-      active: false
-      sourceComponent: AstInspector {
-        anchors.centerIn: Overlay.overlay
-        modelSource: documentModel.debugSource
-        astSource: documentModel.astSource
+    Button {  
+      text: "Debug Source"
+      // outlined: true
```

Comment:
Documentation of [Button](https://doc.qt.io/qt-5/qml-qtquick-controls2-button-members.html) control say nothing about `outlined` prop. It also breaks hot reloading (although app runs successfully with it 🤔 ). Should we keep it here?

Reply:
The `outlined` prop is an [extension of Button made by yours truly](https://github.com/ChaosKit/ChaosKit/blob/master/ui/ChaosKit/Button.qml#L10), which in hindsight was a terrible idea. One additional reason for its terribleness is that it's impossible to switch themes because, well, they don't define it.

A proper solution to this problem would be to create a custom component, `OutlinedButton`, that implements the outlined style.

Reply:
To actually answer your question, I think it's better to use the outlined style there because it doesn't grab your attention as much as the regular style. However, I think it's OK to switch back as a next step.

Reply:
Yeah, agree that outlined style is better for such buttons, let me try to implement it.